doc/ref/types.xml

```diff
@@ -568,7 +575,8 @@ of the object <A>obj</A> is known.
 <Description>
 For an attribute <A>attr</A>, <C>Setter(<A>attr</A>)</C> 
 is called automatically when the attribute value has been
-computed for the first time.
+computed for an immutable object which does not already have stored
+value for <A>attr</A>.
```

Comment:
There's a missing article here, I think 'have a value stored' would make it right.

Reply:
Fixed, thanks.